css/desktop.css

```diff
@@ -0,0 +1,130 @@
+@media (min-width: 786px) {
+  .navbar .hamburger {
+    display: none;
```

Comment:
The `display` property with `none` value is repeated, you could instead create a class for it once and add that class whee necessary in your `html`.